src/System/Wlog/Terminal.hs

```diff
@@ -50,23 +51,31 @@ initTerminalLogging :: MonadIO m
                     -> Bool  -- ^ Show time?
                     -> Bool  -- ^ Show ThreadId?
                     -> Maybe Severities
+                    -> Maybe Severities
                     -> m ()
 initTerminalLogging
     timeF
     customConsoleAction
     isShowTime
     isShowTid
-    (fromMaybe (warningPlus) -> defaultSeverity)
+    maybeSevOut
+    maybeSevErr
   = liftIO $ do
     lock <- liftIO $ newMVar ()
+    let (severitiesOut, severitiesErr) =
+          case (maybeSevOut, maybeSevErr) of
+              (Nothing, Nothing)   -> (excludeError debugPlus, errorPlus)
+              (Just out, Nothing)  -> (out, emptySeverities)
+              (Nothing, Just err)  -> (emptySeverities, err)
+              (Just out, Just err) -> (out, err)
```

Comment:
Am I right that to redirect all output to `stderr` user needs to pass `Nothing` as `maybeSevOut` and `Just allSeverities` as `maybeSevErr`?

Reply:
yes, true. Ideally that would be this line in the `.yaml`:
```
termSeveritiesErr: All
```